frontend/catalyst/__init__.py

```diff
@@ -66,7 +66,6 @@
 )
 
 from catalyst import debug
-from catalyst.ag_utils import AutoGraphError, autograph_source
```

Comment:
Ah, we will need to bring back this top level import to enable `autograph_source` to be documented.